Automattic/wp-calypso
# Toggle flag to remove store in dev, desktop and wpcalypso

#### Changes proposed in this Pull Request

Implements #49428

This toggles off the store in the development, desktop development, and wpcalypso environments. It also simplifies the logic of the deprecation notice and ensures that the store-removed version of the notice is displayed.

#### Testing instructions

1. Check out this branch
2. Rebuild - `yarn start`
3. Disable nav unification - `?disable-nav-unification`
3. Make sure that "Store" isn't displayed for WooCommerce stores.
4. Go to the old store URL - `http://calypso.localhost:3000/store/___.wpcomstaging.com` - and make sure the store removed notice is displayed:

![image](https://user-images.githubusercontent.com/224531/107917852-4d809180-6fb4-11eb-97da-dc9c37de9020.png)
